pkg/chain/local/tbtc.go

```diff
@@ -185,16 +186,16 @@ func (tlc *TBTCLocalChain) CreateDeposit(
 	var utxoValueBytes [8]byte
 	copy(utxoValueBytes[:], utxoValueBytesSlice)
 	fundedAt, _ := new(big.Int).SetString(defaultFundedAt, 0)
-	utxoOutpoint, _ := hex.DecodeString(defaultUtxoOutpoint)
 
 	tlc.deposits[depositAddress] = &localDeposit{
 		keepAddress: keepAddress.Hex(),
 		state:       chain.AwaitingSignerSetup,
 		utxoValue:   big.NewInt(defaultUTXOValue),
 		fundingInfo: &chain.FundingInfo{
-			UtxoValueBytes: utxoValueBytes, // 0x0065cd1d00000000
-			FundedAt:       fundedAt,
-			UtxoOutpoint:   utxoOutpoint, // 0xc27c3bfa8293ac6b303b9f7455ae23b7c24b8814915a6511976027064efc4d5101000000
+			UtxoValueBytes:  utxoValueBytes, // 0x0065cd1d00000000
+			FundedAt:        fundedAt,
+			TransactionHash: previousTransactionHashHex,
+			Index:           previousTransactionIndex,
```

Comment:
We need to test the conversion that is done by `FundingInfo` function in `pkg/chain/ethereum/tbtc.go`. Maybe modifying this mock would be too complex, in such case we could add a `pkg/chain/ethereum/tbtc_test.go`  that would cover testing of slicing and byte reversing the `UtxoOutpoint` returned by the ethereum chain.

Reply:
It seems very difficult to properly mock all of the stuff I'd need in order to get a deposit from the chain, and then get the funding info from that deposit. What if I instead pulled the transaction hash / OutputIndex parsing out into a helper function and then unit tested that?

Reply:
It sounds very reasonable.